### ISSUE TITLE
[kermit-bugsnag] Bugsnag symbol(s) not found for architecture arm64

### ISSUE THREAD

**Issue by Orgmir** [touchlab/Kermit#268]
Not sure if this is an actual issue, or just a clarification needed on the README when setting up Kermit with Bugsnag, but I found myself having to add the `kermit-bugsnag-test` stubs to the `iosMain` dependencies.

I was getting the following error when running `./gradlew :shared:linkDebugFrameworkIosArm64`:
```
The /Applications/Xcode-13.4.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld command returned non-zero exit code: 1.
output:
Undefined symbols for architecture arm64:
  "_OBJC_CLASS_$_Bugsnag", referenced from:
      objc-class-ref in result.o
ld: symbol(s) not found for architecture arm64
```

In my head this makes sense, because I have `kermit-bugsnag` as a dependency in my shared code, but I don't have any actual Bugsnag libraries to link to when linking for Native. Since I want to use library that I declared in my iOS App, I went ahead and added the stubs just so it can link (which is kinda what it says on the `kermit-bugsnag-test` README).

Is this an issue on `kermit-bugsnag` not providing the necessary stubs? Or is this a setup mistake on my part, and I should be adding Bugsnag as a dependency to shared code (which it would be cool if we could clarify this in the README)?

**Comment by Orgmir** [comment]
Probably I should not be doing this, since now I get this error on app start:

```
objc[4634]: Class Bugsnag is implemented in both /private/var/containers/Bundle/Application/EB668D5F-6FF9-4C5E-AE93-3E402F090791/MyApp.app/Frameworks/shared.framework/shared (0x1060a8a68) and /private/var/containers/Bundle/Application/EB668D5F-6FF9-4C5E-AE93-3E402F090791/MyApp.app/MyApp (0x102b1ec68). One of the two will be used. Which one is undefined.
```

**Comment by psh** [comment]
You're correct in the assumption that there are no Bugsnag libraries at link time, but rather than pulling in the stubs you just need to let the linker know to ignore those unresolved symbols.  You can do that directly by passing linker-flags in gradle but we have a plugin that will do that for you

```
plugins {
  id("co.touchlab.crashkios.bugsnaglink") version "x.y.z"
}
```
(See https://github.com/touchlab/CrashKiOS#linking-1)
